PWGLF/TableProducer/converters/v0sconverter.cxx

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/AnalysisDataModel.h"
+#include "PWGLF/DataModel/LFStrangenessTables.h"
+
+using namespace o2;
+using namespace o2::framework;
+
+// Converts V0 version 001 to 002
+struct v0sconverter {
+  Produces<aod::DauTrackExtras_001> dauTrackExtras_001;
+  Produces<aod::V0MCCores_001> v0MCCores_001;
+
+  void process000to001(aod::DauTrackExtras_000 const& dauTrackExtras_000, aod::V0MCCores_000 const& v0MCCores_000)
```

Comment:
This approach will only work to update AO2Ds that are MC, since your process function cannot be used correctly in real data (as real data AO2Ds do not have `V0MCCores` stored). I am afraid you have to separate into two process functions that can be turned on and off independently...